.circleci/config.yml

```diff
@@ -70,8 +70,12 @@ jobs:
     <<: *defaults
     working_directory: /home/fossa/go/src/github.com/fossas/fossa-cli
     docker:
-      - image: quay.io/fossa/fossa-cli-test:circleci
+      - image: quay.io/fossa/fossa-cli-test-base:circleci
     steps:
+      - checkout
+      - run: 
+          name: Compile fossa binary
+          command: make
```

Comment:
If we're compiling here, let's get rid of all the workspace steps that load the compiled binary from the previous step.